graph/src/components/store.rs

```diff
@@ -121,17 +131,18 @@ pub struct EntityQuery {
 }
 
 impl EntityQuery {
-    pub fn new(subgraph_id: SubgraphDeploymentId, entity_types: Vec<String>) -> Self {
+    pub fn new(
+        subgraph_id: SubgraphDeploymentId,
+        entity_types: Vec<String>,
+        range: EntityRange,
```

Comment:
Could we make that `Option<EntityRange>` so callers don't have to create their own all the time?

Reply:
This has two callers and each uses a different range.